analysis/runtime/src/mir_loc.rs

```diff
@@ -47,39 +14,34 @@ pub enum MirProjection {
 /// See [`rustc_middle::mir::Local`](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/mir/struct.Local.html).
 #[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Serialize, Deserialize)]
 pub struct Local {
-    /// TODO(kkysen) change to u32 like
     /// [`rustc_middle::mir::Local`](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/mir/struct.Local.html),
     /// but need to keep bincode binary format.
```

Comment:
This comment doesn't make a lot of sense without its first line--remove entirely/redocument?